inc/3rd-party/plugins/wp-offload-s3-assets.php

```diff
@@ -0,0 +1,39 @@
+<?php
+defined( 'ABSPATH' ) || die( 'Cheatin&#8217; uh?' );
+
+if ( is_admin() && function_exists( 'as3cf_assets_init' ) ) :
+	add_action( 'aws_init', 'rocket_as3cf_assets_compatibility', 13 );
+	add_action( 'update_option_as3cf_assets', 'rocket_maybe_deactivate_cdn', 10, 2 );
+endif;
+
+/**
+ * Compatibility with WP Offload S3 assets addon.
+ * 
+ * @since 2.10.7
+ * @author Remy Perona
+ */
+function rocket_as3cf_assets_compatibility() {
+	global $as3cf_assets;
```

Comment:
OK, understood, it's the plugin "WP Offload S3 - Assets Addon", hooked on `aws_init` with priority 12.
```php
global $as3cf_assets;
...
$as3cf_assets = new Amazon_S3_And_CloudFront_Assets( __FILE__, $aws );
```